lightning/src/ln/channel.rs

```diff
@@ -2330,23 +2330,30 @@ impl<Signer: Sign> Channel<Signer> {
 		stats
 	}
 
-	/// Get the available (ie not including pending HTLCs) inbound and outbound balance in msat.
+	/// Get the available (ie not including pending HTLCs) inbound and outbound balance, plus the
+	/// amount available for a single HTLC send, all in msat.
 	/// Doesn't bother handling the
 	/// if-we-removed-it-already-but-haven't-fully-resolved-they-can-still-send-an-inbound-HTLC
 	/// corner case properly.
 	/// The channel reserve is subtracted from each balance.
 	/// See also [`Channel::get_balance_msat`]
-	pub fn get_inbound_outbound_available_balance_msat(&self) -> (u64, u64) {
+	pub fn get_inbound_outbound_available_balance_msat(&self) -> (u64, u64, u64) {
```

Comment:
this is getting a bit unwieldy. Tuples are great, but I think we may wanna introduce a custom response struct here with field labels.

Reply:
Good idea! Went ahead and moved `get_balance_msat` into the same function while we're at it (and left it as a separate commit for that reason).